sirepo/resource.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+u"""Functions for accessing resource files
+
+:copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+from pykern import pkio
+from pykern import pkresource
+import os
+import sirepo.feature_config
+import werkzeug.utils
+
+
+def filename(*paths):
+    """Get absolute path to a resource file
+
+    Args:
+        paths (str): Names of paths to get to resource file
+
+    Returns:
+        py.path: absolute path to resource file
+    """
+    return pkio.py_path(pkresource.filename(
+        os.path.join(*paths),
+        additional_packages=sirepo.feature_config.dynamic_sim_type_packages(),
```

Comment:
This is repeated too many times. There should be a iterable (tuple) `search_path` that can be passed in to pkresource and importlib. For sirepo, it would add sirepo to the end of the search path, instead of it could be implicitly added in pkresource, but I also think there's nothing wrong with it being explicit, e.g. feature_config.package_path, if used, would have to be set to `('rsprivate','sirepo')` for rsprivate.

Reply:
58607069 is this what you mean?